src/App.js

```diff
@@ -1,23 +1,17 @@
-import logo from './logo.svg';
+import Map from './components/Map';
 import './App.css';
 
 function App() {
+  const location = {
+    lat: 21.694,
+    lng: 71.7979,
+  };
+
+  const zoomLevel = 8;
```

Comment:
We may need to adjust this once we have pins rendering, but it’s a good guess as a starting point!

Reply:
Yeah it's zoomed out fairly far, but we can adjust as we go.